src/tests/end2end/end2end.cpp

```diff
@@ -35,26 +35,14 @@
 #include <sstream>
 #include <fstream>
 #include <vpnes/vpnes.hpp>
-#include <vpnes/gui/config.hpp>
+#include <vpnes/config/config.hpp>
 #include <vpnes/core/frontend.hpp>
 #include <vpnes/core/config.hpp>
 
 /**
  * Configuration for end-to-end testing
  */
-struct SConfig : vpnes::gui::SApplicationConfig {
-protected:
-	/**
-	 * Parse command line option
-	 *
-	 * @param name Option name
-	 * @param value Argument
-	 * @return Valid option or not
-	 */
-	bool parseOption(const std::string &name, const std::string &value) {
-		return vpnes::gui::SApplicationConfig::parseOption(name, value);
-	}
-
+struct SConfig : vpnes::config::SApplicationConfig {
```

Comment:
This class is supposed to define some options for testing. It doesn't seem to be implemented